README.md

```diff
@@ -1,9 +1,89 @@
 # stocktrader
 
+[![GitHub license](https://img.shields.io/github/license/whzinformatik/stocktrader)][license]
+[![GitHub release](https://img.shields.io/github/v/release/whzinformatik/stocktrader)][release]
+
+The project contains various projects to implement a stock trader. A stock trader should be able to buy and sell shares. Various small projects have been developed for this purpose. General information on the entire project is described on the following page. Further information can be found in the respective sub-project.
+
 ## Service Architecture
 
+The following graphic roughly describes the architecture of the whole project:
+
+[![ContextMap][contextMap]][contextMap]
+
+The services can be accessed via the following default ports:
+
 - feedback: port 18080
 - comment tone: port 18081
 - portfolio: port 18082
 - stock quote: port 18083
 - account: port 18084
+
+## Prerequisites
+
+The project is created with the build tool [Maven][maven], [Docker][docker] and the [Java Version 11][openjdk].
+
+## Getting Started
+
+If you want to run services locally, please follow the instructions in the respective projects. You can set up all other important services with Docker:
+
+```bash
+docker-compose up -d TODO
+```
```

Comment:
@simeck26 When you are done with Docker, I would still adjust the line. I would also leave the PR open until then.